nuxeo-runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ConfigurationGenerator.java

```diff
@@ -375,6 +385,24 @@ protected Properties getStoredConfig() {
         }
     };
 
+    public static File[] getJarFilesFromPattern(File dir, String pattern) {
+        final Pattern jarPattern = Pattern.compile(pattern);
```

Comment:
Remove redundant `final`, the variable is already _effectively final_ which is enough for the closure.